app/models/recommender.rb

```diff
@@ -8,6 +9,27 @@ def initialize(question, user = nil)
   end
 
   def recommendation
-    @choices.sample
+    max_hash(rate_choices).first
+  end
+
+  def rate_choices
+    results = {}
+    choices.each do |choice|
+      query_string = search_choice(question, choice)
+      search_results = execute_search(query_string, user)
+      choice_ratings = search_results.map(&:rating)
+      results[choice] = average(choice_ratings)
+    end
+    results
+  end
+
+  def max_hash(hash)
+    hash.max_by { |choice, rating| rating }
```

Comment:
Unused block argument - `choice`. If it's necessary, use `_` or `_choice` as an argument name to indicate that it won't be used.